app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+
+  def new
+    @image = Image.find(params[:image_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @image = Image.find(params[:image_id])
+    @comment = @image.comments.new(comment_params)
```

Comment:
This is an example of just making Rails work, whilst forgetting the basics we learnt in the first few weeks.

In Boris Bikes we didn't do:

`dock.bikes << bike`

`dock.add_bike(bikes)` or `dock << bike`.

Why? Firstly we want to encapsulate our logic inside of the dock object. Secondly, [tell don't ask](https://robots.thoughtbot.com/tell-dont-ask).

So instead we would do:

```
@image.add_comment(comment_params)
```

Then we would unit test this method, which would exist in the model:

```
class Image < ActiveRecord::Base
  def add_comments(comment_params)
    comments.new(comment_params)
  end
```